src/main/java/Rabbit/RabbitException/FindFormatException.java

```diff
@@ -0,0 +1,8 @@
+package Rabbit.RabbitException;
+
+public class FindFormatException extends RabbitException {
+    @Override
+    public String toString() {
+        return "Hmmm...I don't get what you're trying to find. Human language please?";
```

Comment:
I really like how comprehensive all your Exceptions are! A small consideration to make: perhaps, adding more details to the printed message here might help highlight the difference between what the user must do after getting a `FindFormatException` and an `InvalidInputException`.